Libplanet.Explorer.Executable/Program.cs

```diff
@@ -370,6 +371,7 @@ private static BlockPolicy<T> LoadBlockPolicy<T>(Options options)
                 await swarm.BootstrapAsync(
```

Comment:
How about using newly introduced `Swarm<T>.BootstrapAsync(CancellationToken)` here?

Reply:
Done. See https://github.com/planetarium/libplanet/pull/2024/commits/7e5fe076f5fb24593319d8d7c825bd2b8559ffd3. We probably need to overhaul `Program.cs` at some point. 🙃